packages/platform/types/event-types/event-types_2024_06_14/inputs/update-event-type.input.ts

```diff
@@ -262,11 +317,16 @@ export class UpdateEventTypeInput_2024_06_14 {
     return value;
   })
   @ValidateNested()
+  @DocsPropertyOptional({
+    description: "Create a recurring event type.",
```

Comment:
needs an update
same for `create-event-type.input.ts`

Reply:
Fixed in [refactor: seats description](https://github.com/calcom/cal.com/pull/16854/commits/c5dde562abde0703e56ea01ffa70543fc42529fb)